cli/command/container/cp.go

```diff
@@ -272,13 +333,33 @@ func copyToContainer(ctx context.Context, dockerCli command.Cli, copyConfig cpCo
 
 		resolvedDstPath = dstDir
 		content = preparedArchive
+		if !copyConfig.quiet {
+			content = &copyProgressPrinter{
+				ReadCloser:  content,
+				toContainer: true,
+				writer:      dockerCli.Err(),
+				total:       &copiedSize,
+			}
+		}
 	}
 
 	options := types.CopyToContainerOptions{
 		AllowOverwriteDirWithFile: false,
 		CopyUIDGID:                copyConfig.copyUIDGID,
 	}
-	return client.CopyToContainer(ctx, copyConfig.container, resolvedDstPath, content, options)
+
+	if !copyConfig.quiet {
+		return client.CopyToContainer(ctx, copyConfig.container, resolvedDstPath, content, options)
+	}
+
+	fmt.Fprint(dockerCli.Err(), "Preparing to copy...\n")
+	fmt.Fprint(dockerCli.Err(), "\033[s")
```

Comment:
🤦 @maximillianfx I missed updating this one

```suggestion
	fmt.Fprint(dockerCli.Err(), aec.Save)
```

Reply:
Okay 😄, I'll update this.

Reply:
Fixed.